style.css

```diff
@@ -83,4 +85,277 @@
     font-style: normal;
     margin-top: 10px;
   }
+
+  .card-id {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    margin-top: 50px;
+    gap: 20px;
+  }
+
+  .card-one {
+    margin-top: 20px;
+    background-color: white;
+    width: 327px;
+    height: 552px;
+    border-radius: 16px;
+    margin-bottom: 30px;
+    transition: all 0.5 ease;
+  }
+
+  .card-one img {
+    margin-top: 20px;
+  }
+
+  .card-one ul {
+    display: inline-flex;
+    gap: 10px;
+    position: relative;
+  }
+
+  .card-one ul li {
+    font-size: 12px;
+    font-weight: 500;
+    color: #344563;
+    position: relative;
+    left: -34px;
+  }
+
+  .card-one ul li h4 {
+    width: 55px;
+    height: 16px;
+    font-style: normal;
+    font-weight: 600;
+    font-size: 13px;
+    line-height: 16px;
+    color: #344563;
+  }
+
+  .card-one ul li img {
+    position: relative;
+    top: -14px;
+  }
+
+  .card-one h3 {
+    width: 295px;
+    height: 36px;
+    font-style: normal;
+    font-weight: 700;
+    font-size: 20px;
+    line-height: 44px;
+    color: #172b4d;
+    position: relative;
+    left: -38px;
+    margin-bottom: 10px;
+  }
+
+  .card-one .tonic {
+    color: #172b4d;
+    position: relative;
+    left: -93px;
+  }
+
+  .textTonic p {
+    width: 295px;
+    height: 76px;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 15px;
+    line-height: 24px;
+    color: #344563;
+    flex: none;
+    order: 1;
+    flex-grow: 0;
+    text-align: start;
+    position: relative;
+    left: +20px;
+  }
+
+  .language ul li {
+    display: flex;
+    flex-direction: row;
+    justify-content: center;
+    padding: 3px 13px;
+    background: #ebebff;
+    border-radius: 8px;
+    color: #6070ff;
+    position: relative;
+    left: -39px;
+    margin-top: 10px;
+  }
+
+  .seeMore h3 {
+    /* Button /Normal */
+    font-style: normal;
+    font-weight: 500;
+    font-size: 17px;
+    line-height: 24px;
+    display: flex;
+    position: relative;
+    left: 13px;
+    margin-top: 20px;
+    align-items: center;
+    padding: 4px 5px;
+    gap: 12px;
+
+    /* Primary/B500 */
+    border: 1px solid #396df2;
+    width: 123px;
+    color: #396df2;
+    border-radius: 8px;
+    transition: all 0.5s ease;
+  }
+
+  .seeMore h3:hover {
+    background: #396df2;
+    color: white;
+  }
+
+  footer {
+    position: absolute;
+    width: 375px;
+    height: 1263px;
+    background: #fff;
+    border-top-right-radius: 120px;
+  }
+
+  .aboutme {
+    margin-top: 100px;
+  }
```

Comment:
- [ ] Dear @tchasinga, as part of the main requirements for the project the content of the `about me` section is expected to be aligned with Flexbox. This is to make sure you master the skill of using different alignments in your website.